tests/test_examples.py

```diff
@@ -32,6 +35,25 @@ def test_examples_1(self):
             preprocessed_data=data,
         )
         verif.run()
+        end_time_single_thread = time.time()
+
+        # Run example (multi thread)
+        start_time_multi_thread = time.time()
+        verif.configure(
+            output_path="./",
+            lib_items_path=ex.library(),
+            plot_option="all-expand",
+            fig_size=(10, 5),
+            num_threads=3,
+            preprocessed_data=data,
+        )
+        verif.run()
+        end_time_multi_thread = time.time()
+        assert (end_time_single_thread - start_time_single_thread) > (
```

Comment:
Test to check that running multiple threads at once is quicker than on a single thread.